CMakeLists.txt

```diff
@@ -4,7 +4,8 @@ project(yapvm)
 
 set(CMAKE_CXX_STANDARD 23)
 set(CMAKE_CXX_STANDARD_REQUIRED ON)
-             
+add_compile_options(-fsanitize=address)
```

Comment:
Not portable in case of MSVC in example, create new target (with name "ASAN" i.e.) for compile and run with sanitizers